.circleci/config.yml

```diff
@@ -25,3 +25,124 @@ jobs:
           command: pipenv run mkdocs build --clean --strict --verbose
       - store_artifacts:
           path: site
+
+# Auto changelog collector
+  github-changelog-generator:
+    working_directory: ~/build
+    docker:
+      - image: ferrarimarco/github-changelog-generator:1.14.3
+    steps:
+      - setup_remote_docker:
+           version: 17.11.0-ce
+      - checkout
+      - run:
+           name: Build changelog
+           working_directory: ~/build
+           command: |
+             if git log -1 --pretty=%s | grep Merge* ; then
+             if git log -1 --pretty=%b | grep release ; then
+             echo "New release, do nothing"
+             else
+             github_changelog_generator --user bids-standard --project bids-specification --token ${CHANGE_TOKEN} --output ~/build/CHANGES.md --base ~/build/src/pregh-changes.md --header-label Changelog --no-issues --no-issues-wo-labels --no-filter-by-milestone --no-compare-link --pr-label ""
+             cat ~/build/CHANGES.md
+             mv ~/build/CHANGES.md ~/build/src/CHANGES.md
+             fi
+             else
+             echo "Not a PR merge commit"
+             fi
+      - persist_to_workspace:
+             root: .
+             paths: src
+
+# Clear remark test
+  remark:
+    working_directory: ~
+    docker: 
+      - image: node:latest 
+    steps: 
+      - checkout 
+      - attach_workspace:
+          at: ~/build
+      - run:
+          name: update-npm
+          command: |
+           cd ~
+           npm install npm@latest
+      - run: 
+          name: get remark
+          command: |
+           cd ~
+           npm install remark remark-cli
+      - run: 
+          name: get remark styles
+          command: |
+           cd ~
+           npm install remark-cli@5.0.0 remark-lint@6.0.2 remark-preset-lint-recommended@3.0.2 remark-preset-lint-markdown-style-guide@2.1.2
+      - run: # remark the auto generated changes.md
+          name: remark on autogenerated CHANGES.md
+          command: |
+             cd ~/project
+             if git log -1 --pretty=%s | grep Merge* ; then
+             if git log -1 --pretty=%b | grep release ; then
+             echo "New release, do nothing"
+             mkdir ~/project/src/tmp
+             touch ~/project/src/tmp/empty.txt
```

Comment:
This seems identical to the "not a pr merge commit" path - logic could be simplified.

Reply:
Across the two jobs it is a similar path. The jobs have to be separated because of the docker environment differences between the two jobs.